cli/dts/lib.deno.shared_globals.d.ts

```diff
@@ -712,7 +721,18 @@ declare class Worker extends EventTarget {
        * hello world2
        *
        */
-      deno?: boolean;
+      deno?: {
+        namespace?: boolean;
+        permissions?: "inherit" | {
+          env?: "inherit" | boolean;
+          hrtime?: "inherit" | boolean;
+          net?: "inherit" | boolean | string[];
```

Comment:
A bit strange that `read` and `write` accept `URL` but `net` doesn't

Reply:
Maybe docs should be more explicit here but these are the correct types. Net allowlists should be a list of hosts (`"<hostname>[:<port>]"`) not representable with URLs.

Reply:
Might be a little overkill, but I added docs specific to that field to specify the format of the URLs. Just in case